webapp/components/post_view/components/post_list.jsx

```diff
@@ -392,15 +392,15 @@ export default class PostList extends React.Component {
         if (this.props.scrollType === ScrollTypes.BOTTOM) {
             this.scrollToBottom();
         } else if (this.props.scrollType === ScrollTypes.NEW_MESSAGE) {
-            window.setTimeout(window.requestAnimationFrame(() => {
+            window.requestAnimationFrame(() => {
```

Comment:
Also I don't think it will call it 60 times per second unless you add recursion in, it should just perform it at the next available opportunity
https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame